etc/bash_completion.d/juju-2.0

```diff
@@ -42,9 +42,11 @@ import json, sys
 sys.stderr.close()
 j = json.load(sys.stdin)
 all_units = []
-for k, v in j.get("applications", {}).items():
-    all_units.extend(v.get("units", {}).keys())
-print ("\n".join([unit + trail for unit in all_units]))
+try:
+    for k, v in j.get("applications", {}).items():
+        all_units.extend(v.get("units", {}).keys())
+    print ("\n".join([unit + trail for unit in all_units]))
+except: pass
```

Comment:
I think we don't want `try:/except: pass` - yes the kipple when things break is bad, but things being broken with no feedback as to why is worse I feel.